### ISSUE TITLE
[ERROR] [toolbar] Failed to load dev toolbar app from @storyblok/astro/toolbar-app-storyblok.ts: Failed to resolve module specifier '@storyblok/astro/toolbar-app-storyblok.ts'

### ISSUE THREAD

**Issue by hamiltonbrooke** [storyblok/storyblok-astro#645]
Keep getting the following error in my console:

_[ERROR] [toolbar] Failed to load dev toolbar app from @storyblok/astro/toolbar-app-storyblok.ts: Failed to resolve module specifier '@storyblok/astro/toolbar-app-storyblok.ts'_

**Comment by MDmubarak786** [comment]
I am also facing this issue

**Comment by s-niwas** [comment]
I also encounter the same error, which negatively impacts my performance score.
<img width="1120" alt="Screenshot 2024-01-26 at 4 52 52 PM" src="https://github.com/storyblok/storyblok-astro/assets/137252828/077e5614-3555-4a84-bc78-3e5338495a28">

**Comment by Justin-Arnold** [comment]
I am also having this same issue, appeared when after moving from astro ~4.0.0 to ~4.2.0

**Comment by schabibi1** [comment]
Thanks everyone for reporting it.
I tried to reproduce it but I cannot reproduce this issue.

Could you all try out on your end by following these steps?
1. `npm create astro@latest`
2. `npm install @storyblok/astro` (Make sure you install 4.0.0 or higher)
3. Add preview API key token in `astro.config.mjs`
4. Preview on the visual editor

Basically, follow this [Ultimate Tutorial part 1](https://www.storyblok.com/tp/add-a-headless-cms-to-astro-in-5-minutes) we have. 
The dev toolbar worked just fine when I tested it. [Screen recording](https://www.loom.com/share/fa4dc4bea0454d0bbd56b77900384964?sid=58e2d92b-cb53-4a30-b835-82921c8b9703)

My colleague also confirmed it worked fine. It could be that the Astro version may impact from what I see the tested version from @Justin-Arnold 
Please let us know if you still face the issue and feel free to re-open this issue with reproduction if you still see the same issue you reported.

- Tested dependencies
```js
  "dependencies": {
    "@astrojs/check": "^0.5.6",
    "@storyblok/astro": "^4.0.2",
    "@vitejs/plugin-basic-ssl": "^1.1.0",
    "astro": "^4.4.5",
    "typescript": "^5.3.3",
    "vite": "^5.1.4"
  }
}
```

**Comment by schabibi1** [comment]
Just in case, I fixed the deprecated method name #671
